example/package.json

```diff
@@ -46,13 +45,13 @@
     "html-webpack-plugin": "^5.5.0",
     "jest": "^26.6.3",
     "metro-react-native-babel-preset": "^0.67.0",
+    "patch-package": "^6.4.7",
```

Comment:
We still need to patch the package to use the `full/` entry point in the package but I feel like this could be fixed by using the `full/` module url directly in `package/` with our own type declaration for that `full` module. then we can remove `patch-package` even in `package`. The only thing we would need to do it have a rule that nowhere we import CanvasKit via "canvaskit-wasm"